py-scripts/lf_ftp.py

```diff
@@ -926,6 +1086,18 @@ def get_device_details(self):
             logger.info("No endpoint found")
             logger.info(total_data)
 
+    # Updates the status in the running.json file while running a test from the Web UI
+    def updating_webui_runningjson(self, obj):
```

Comment:
Since we haven't really had a software architecture discussion across teams on the interaction between WebUI and the scripts it uses, this is fine. I don't love the relative paths here, but I don't have any better suggestions at this time. No change needed.